content/blog/kubernetes-monitoring-using-prometheus-and-grafana-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,419 @@
+pod/prometheus-stack-prometheus-node-exporter-bcmt2            1/1     Running   0          4d17h
+pod/prometheus-stack-prometheus-node-exporter-hgr2x            1/1     Running   0          4d17h
+pod/prometheus-stack-prometheus-pushgateway-559cc996d5-jrjdn   1/1     Running   0          4d17h
+pod/prometheus-stack-server-7646574d75-zswws                   2/2     Running   0          4d17h
+
+NAME                                                TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)        AGE
+service/grafana-dashboard                           NodePort    10.97.7.23       <none>        80:30488/TCP   4d17h
+service/prometheus-stack-alertmanager               ClusterIP   10.101.171.170   <none>        9093/TCP       4d17h
+service/prometheus-stack-alertmanager-headless      ClusterIP   None             <none>        9093/TCP       4d17h
+service/prometheus-stack-kube-state-metrics         ClusterIP   10.98.134.240    <none>        8080/TCP       4d17h
+service/prometheus-stack-prometheus-node-exporter   ClusterIP   10.104.44.251    <none>        9100/TCP       4d17h
+service/prometheus-stack-prometheus-pushgateway     ClusterIP   10.104.146.18    <none>        9091/TCP       4d17h
+service/prometheus-stack-server                     NodePort    10.106.152.143   <none>        80:30245/TCP   4d17h
+
+NAME                                                       DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR            AGE
+daemonset.apps/prometheus-stack-prometheus-node-exporter   2         2         2       2            2           kubernetes.io/os=linux   4d17h
+
+NAME                                                      READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/grafana-dashboard                         1/1     1            1           4d17h
+deployment.apps/prometheus-stack-kube-state-metrics       1/1     1            1           4d17h
+deployment.apps/prometheus-stack-prometheus-pushgateway   1/1     1            1           4d17h
+deployment.apps/prometheus-stack-server                   1/1     1            1           4d17h
+
+NAME                                                                 DESIRED   CURRENT   READY   AGE
+replicaset.apps/grafana-dashboard-5674bcd6d4                         1         1         1       4d17h
+replicaset.apps/prometheus-stack-kube-state-metrics-6fb8684695       1         1         1       4d17h
+replicaset.apps/prometheus-stack-prometheus-pushgateway-559cc996d5   1         1         1       4d17h
+replicaset.apps/prometheus-stack-server-7646574d75                   1         1         1       4d17h
+
+NAME                                             READY   AGE
+statefulset.apps/prometheus-stack-alertmanager   1/1     4d17h
+```
+
+T﻿yping *helm list* command, it will show both Prometheus and Grafana helm charts and versions that are deployed to the *monitoring* namespace in the cluster:
```

Comment:
Change to:
"Type the help list command to show both Prometheus and Grafana help charts and versions that are deployed ..."